script_test.go

```diff
@@ -1768,6 +1769,57 @@ func TestWithStdout_SetsSpecifiedWriterAsStdout(t *testing.T) {
 	}
 }
 
+func TestWithEnv_SetEmptyEnvironment(t *testing.T) {
```

Comment:
Could we be a bit more descriptive here?

```suggestion
func TestWithEnv_UnsetsAllEnvVarsGivenEmptySlice(t *testing.T) {
```